README.md

```diff
@@ -1,81 +1,172 @@
 # decss
 
-CSS components
+decss converts [CSS modules] to React/Preact components. Thanks to
```

Comment:
I wanted to highlight an important term that people could search for. I'll make it consistent.